doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
+//approved
+.In a high-availability authoring environment, project creation fails when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2475[RHPAM-2475]]
 
-Workaround: None.
+Issue: When you create a project, the project creation does not complete. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
 
-ifdef::DM[]
+Workaround: Create the project again.
 
 //approved
-.Users cannot test DMN context expression in test scenarios [https://issues.jboss.org/browse/RHDM-1116[RHDM-1116]]
+.In a high-availability authoring environment, asset creation is not completed when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2476[RHPAM-2476]]
+
+Issue: When you create an asset, the asset is not indexed. It is displayed in {CENTRAL} but you cannot use it. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
+
+Workaround: In the *Project Explorer* view, open the asset and click *Save*.
+
+== DMN designer
 
-Issue: If you test DMN context expressions using the test scenario designer and the test fails, the corrected output has the wrong format.
+.When importing a data object from a Java class, a field of the Java Date type is not converted to the DMN date type [https://issues.redhat.com/browse/RHDM-1145[RHDM-1145]]
+
+Issue: When you use DMN designer to import a data object from a Java class and the class has a field of type Date, the resulting DMN field is not of a DMN date type.
 
 Steps to reproduce:
 
-* Create two tests where the second test is created according to the hint message in the first test, while retaining the same input as the first test.
+. Open a project in {CENTRAL}.
+. Add a Java data object.
+. Create a `java.util.Date` field in this object.
+. Save and close the Java class.
+. Open a DMN file.
+. Select the *Data Types* tab.
+. Click *Import Data Object*.
+. Select the Java class that you created.
+. Click *Import*.
+. Check the type of the field into which the `java.util.Date` field was converted.
 
-Expected result: The corrected output has the correct format, as shown in the following example:
+Expected result: The field has the type `date` or `date and time`.
 
-`{"key_a" : "value_a", "key_b" : "value_b", ...}`
+Actual result: The field has the type `Any`.
 
-Actual result: The corrected output has an incorrect format (missing double quotes), as shown in the following example:
+Workaround: Manually change the type of the field to `date` or `date and time`.
 
-`{key_a : value_a, key_b : value_b, ...}`
+.When importing a data object from a Java class, a field of the Java List type is not converted to the DMN collection type [https://issues.redhat.com/browse/RHDM-1144[RHDM-1144]]
 
+Issue: When you use DMN designer to import a data object from a Java class and the class has a field of type List, the resulting DMN field is not of a DMN collection type.
```

Comment:
`List`, the  DMN field does not have the DMN `collection` type.
List and collection need code formatting.